api/NavigatorLanguage.json

```diff
@@ -0,0 +1,292 @@
+                }
+              ],
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "11",
+                "notes": "Closest available (non-standard) properties are <code><a href='http://msdn.microsoft.com/en-us/library/ie/ms534713.aspx'>userLanguage</a></code> and <code><a href='http://msdn.microsoft.com/en-us/library/ie/ms533542.aspx'>browserLanguage</a></code>."
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "on_workernavigator": {
+            "__compat": {
+              "description": "on WorkerNavigator",
+              "support": {
+                "webview_android": {
+                  "version_added": null
+                },
+                "chrome": {
+                  "version_added": null
+                },
+                "chrome_android": {
+                  "version_added": null
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "35"
+                },
+                "firefox_android": {
+                  "version_added": "35"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": null
+                },
+                "opera_android": {
+                  "version_added": null
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              }
+            }
+          }
+        },
+        "languages": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/NavigatorLanguage/languages",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": "32",
+                "notes": "In Chrome, <code>navigator.language</code> is the language of the browser UI, and is not guaranteed to be equal to <code>navigator.languages[0]</code>."
```

Comment:
Copy this to `webview_android` and `chrome_android`.

Reply:
👍